windows/setup-ssh.ps1

```diff
@@ -8,3 +8,4 @@ Start-Service sshd
 Get-Service -Name sshd | Set-Service -StartupType Automatic
 New-NetFirewallRule -Name sshd -DisplayName 'OpenSSH Server (sshd)' -Enabled True -Direction Inbound -Protocol TCP -Action Allow -LocalPort 22
 New-ItemProperty -Path "HKLM:\SOFTWARE\OpenSSH" -Name DefaultShell -Value "C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe" -PropertyType String -Force
+wmic UserAccount set PasswordExpires=False
```

Comment:
@filipleple I do not think this script is used in the autoinstaller. The script is rather embedded in the XML.

Reply:
As of https://github.com/Dasharo/preseeds/commit/4c0f692a461258ea7ebc69a2273ec8c11f7898da all commands are executed in the .xml and in the .ps1 script as fallback